frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewSidePanel/FeatureOverviewSidePanelDetails/useShowDependentFeatures.ts

```diff
@@ -5,5 +5,5 @@ export const useShowDependentFeatures = (project: string) => {
     const { dependenciesExist } = useCheckDependenciesExist(project);
```

Comment:
Should the project truthy check be prior to this as well? Or is that covered by the conditionalSWR in `useCheckDependenciesExist`?

Reply:
conditional one verifies project inside the hook